datasquare_test.go

```diff
@@ -420,6 +422,54 @@ func BenchmarkEDSRoots(b *testing.B) {
 	}
 }
 
+func BenchmarkEDSRootsWithErasuredNMT(b *testing.B) {
+	ODSSizeByteUpperBound := 512 * 1024 * 1024 // converting 512 MiB to bytes
+	totalNumberOfShares := float64(ODSSizeByteUpperBound) / shareSize
+	// the closest power of 2 of the square root of
+	// the total number of shares
+	nearestPowerOf2ODSSize := math.Pow(2, math.Ceil(math.Log2(math.Sqrt(
+		totalNumberOfShares))))
+	namespaceIDSize := 29
+
+	for i := 32; i <= int(nearestPowerOf2ODSSize); i *= 2 {
+		// number of shares in the original data square's row/column
```

Comment:
[optional] could replace `i` with `squareSize` (or `odsSize` if you prefer)

Reply:
I incorporated your suggestion [a67997e](https://github.com/celestiaorg/rsmt2d/pull/304/commits/a67997ec08f9079f99271fd78dbb84af4bea74f2)